web/html/src/build/webpack.config.vendors.js

```diff
@@ -4,15 +4,19 @@ const vendors = require("../vendors/vendors");
 const LicenseCheckerWebpackPlugin = require("license-checker-webpack-plugin");
 const CleanWebpackPlugin = require('clean-webpack-plugin');
 
-function getWebpackConfig({generateLicenses}) {
+module.exports = (env, argv) => {
+
+  const  isProductionMode = argv.mode !== "development";
```

Comment:
All the other checks are like `argv && argv.foo ...` maybe this should be the same?